src/reducers/comments.tsx

```diff
@@ -209,6 +240,38 @@ export const fetchAndLoadComments = ({
   };
 };
 
+export const deleteComment = ({
+  _requestCommentDeletion = requestCommentDeletion,
+  addonId,
+  commentId,
+  versionId,
+}: {
+  _requestCommentDeletion?: typeof requestCommentDeletion;
+  addonId: number;
+  commentId: number;
+  versionId: number;
+}): ThunkActionCreator => {
+  return async (dispatch, getState) => {
+    const { api: apiState } = getState();
+
+    dispatch(actions.beginDeleteComment({ commentId }));
+
+    const response = await _requestCommentDeletion({
```

Comment:
Nit: I understand why it is named like this, but I find this new name "far" from the original name.

Reply:
And that makes the code harder to follow? Or something else? I changed the alias to `apiDeleteComment` which is closer to the original name but the code is a bit less readable to me. In any case, no biggie -- I don't mind changing it.